src/argowrapper/routes/routes.py

```diff
@@ -151,21 +151,26 @@ def check_user_billing_id(request):
     remove gen3 username from pod metadata
     """
 
-    header = {"Authorization", request.headers.get("Authorization")}
-    url = request.base_url._url.rstrip("/") + "/user/user"
+    header = {"Authorization": request.headers.get("Authorization")}
+    url = "http://fence-service/user"
```

Comment:
this is a good default. Let's have a follow-up ticket to make this configurable.